### PR TITLE
Release 6.6.0

### PR DESCRIPTION
**Pull request by briskt**
### Added
- Use Sentry for error logging and reporting
- New pull request template

### Fixed
- Use a specified version of silinternational/serverless-mfa-api-go to fix broken tests. [IDP-750](https://itse.youtrack.cloud/issue/IDP-750)

### CODE REVIEW

File: application/common/config/main.php [hunk 1]
```diff
@@ -189,6 +191,24 @@
...
+                    'enabled' => true,
+                    'dsn' => Env::get('SENTRY_DSN'),
+                    'levels' => ['error'],
+                    'context' => true,
+                    // Additional options for `Sentry\init`
+                    // https://docs.sentry.io/platforms/php/configuration/options
+                    'clientOptions' => [
+                        'attach_stacktrace' => false, // stack trace identifies the logger call stack, not helpful
+                        'environment' => YII_ENV,
+                        'release' => 'idp-id-broker@6.6.0',
```
Comment (forevermatt) [inline application/common/config/main.php:205]: If you want, we can have our CI/CD build process write a file with the current version number in it (from the git tag), and then you could read that file in order to dynamically show the current version number here, without the need to manually update this file with each release.

We do something similar in another project's Dockerfile, though this is to show the build date, not the version number:
```Dockerfile
# Record now as the build date/time (in a friendly format).
RUN date -u +"%B %-d, %Y, %-I:%M%P (%Z)" > /data/protected/data/version.txt
```

To get the git tag, I think we'd just need to add a build arg, to be supplied by Codeship at build time.

Just an idea.

**Review by forevermatt — APPROVED** [review]
Added some comments about minor ideas/suggestions, none of them necessary. Approved.

File: application/common/config/main.php [hunk 2]
```diff
@@ -1,5 +1,7 @@
 <?php
 
+use notamedia\sentry\SentryTarget;
+use Sentry\Event;
```
[the remark this answers is not in the document]
Comment (briskt) [reply application/common/config/main.php:4]: Hmm, I thought I had considered that but obviously not. Just to make sure I get it right the first time, are they sorted by the beginning of the import path or by the last element in the path?

File: application/common/config/main.php (hunk 1 above)
Comment (briskt) [reply application/common/config/main.php:205]: Yeah, I had that thought, but it seems like every time I try to do something like that it takes me numerous attempts to get it right. I was less interested in going through that this time.

File: application/common/config/main.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (forevermatt) [reply application/common/config/main.php]: By the beginning (i.e. by the whole string, not just the last segment)

**Pull request merged**